### PR TITLE
update amplitude instrumentation

### PR DESCRIPTION
**Pull request by dereckquock**
https://docs.google.com/spreadsheets/d/1frV-5H2sb_9H1K_MDsWS-OL-BiicqLy65tKdO3rAEAA/edit#gid=0
https://analytics.amplitude.com/echochurch/manage/project/267356/schema?sort=name&direction=asc

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -8,13 +14,13 @@ import * as Amplitude from 'expo-analytics-amplitude';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import AppNavigator from './navigation/AppNavigator';
 
-// init with the dev app https://analytics.amplitude.com/echo-church-app-dev
 Amplitude.initialize('915eae61254cd5eaec60df02369eff57');
```
Comment (Phillip3) [inline App.js]: @dereckquock should this be in "Keys.js"?

File: README.md [hunk 2]
```diff
@@ -1,62 +1,82 @@
-# The Resonance Project
+# 🔊 The Resonance Project
```
Comment (Phillip3) [inline README.md:1]: Emojis everywhere 😆

File: components/ModalSheet.js [hunk 3]
```diff
@@ -43,15 +49,6 @@ export default ({
         swipeDirection="down"
         style={styles.modal}
       >
-        <AntDesign.Button
```
Comment (Phillip3) [inline components/ModalSheet.js:46]: I think you can remove https://github.com/echo-labs-team/the-resonance-project/blob/7805fc428ae6ab2c3c0b57fac4df2680e35dc777/components/ModalSheet.js#L11

File: screens/Home.js [hunk 4]
```diff
@@ -27,16 +27,14 @@ import Spinner from '../components/Spinner';
...
-  mainTray: 'Home',
-};
-
 const storePostsData = async data => {
   await AsyncStorage.setItem('@posts', JSON.stringify(data)).catch(err =>
     console.error(err)
   );
 };
+const getStoredPosts = () => {
+  return AsyncStorage.getItem('@posts').catch(err => console.error(err));
```
Comment (Phillip3) [inline screens/Home.js:36]: Not tracking this error?

File: screens/Media.js [hunk 5]
```diff
@@ -11,7 +11,6 @@ import {
   View,
 } from 'react-native';
 import { useSafeArea } from 'react-native-safe-area-context';
-import WebView from 'react-native-webview';
```
Comment (Phillip3) [inline screens/Media.js:14]: Good cleanup! Thanks! ✨

File: App.js (hunk 1 above)
Comment (dereckquock) [reply App.js]: yeah 👍  let's put it there

File: screens/Home.js (hunk 4 above)
Comment (dereckquock) [reply screens/Home.js:36]: nawww...it should work 👍  but we don't want it to break anything if it doesn't return any stored data just in case

**Review by Phillip3 — APPROVED** [review]
Lookin' good, good lookin'!

**Pull request merged**